src/utils/resource-utils.js

```diff
@@ -14,31 +15,26 @@ import FileUnknownFilledIcon from '../components/icons/files/file-unknown-filled
 
 const extensionsGroups = [
   { type: RESOURCE_TYPE.pdf, extensions: ['pdf'] },
-  { type: RESOURCE_TYPE.text, extensions: ['txt', 'doc', 'rtf', 'odt'] },
+  { type: RESOURCE_TYPE.text, extensions: ['txt'] },
```

Comment:
There's actually no reason to have a separate `document` category for file types that no browser can do anything with (and why wouldn't we then not also have a category for Powerpoint presentations or Excel sheets?). So everything that's not simply plain text should be `unknown` (can be displayed as "other" on the UI). I will update the ticket accordingly.

Reply:
Indeed, sounds good! 👍